scripts/local-dev/setup-rbe-for-local-dev.sh

```diff
@@ -0,0 +1,63 @@
+#!/bin/bash
+# A script for automatically configuring a user's local dev
+# environment to use Remote Build Execution.
+
+# Determine the root directory of the Angular github repo.
+GITHUB_DIRECTORY_ROOT=$(git rev-parse --show-toplevel 2> /dev/null);
+if [[ $? -ne 0 ]]; then
+  echo "This command must be run from within the angular repository."
+  exit 1;
+fi
+
+# Confirm gcloud installed and available as a command.
+if [ ! -x "$(command -v gcloud)" ]; then
+  echo "gcloud command is not available.  Please install gcloud before continuing."
+  exit 1;
+fi
+
+# Confirm the parameter provided to the script is a directory
+if [[ ! -d $1 ]]; then
+  echo "The paramter provided is not a directory";
```

Comment:
There is a typo for `parameter` here. How about showing the usage instead? Maybe also expand on why we need to even specify a directory.

```suggestion
  echo "Invalid command syntax. Usage: ./script.sh <ServiceAccountKeyLocation>";
```